### PR TITLE
Audio_Init_Params.c Cleanup

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/code/audio_init_params.c [hunk 1]
```diff
@@ -10,79 +10,97 @@ const s16 D_8014A6C0[] = {
...
+#define SFX_SOUNDFONT_1_SIZE 0x3AA0
+#define SFX_SOUNDFONT_2_SIZE 0x17B0
 
-ReverbSettings D_80133420[][3] = {
+// Sizes of everything on the init pool
+#define AI_BUFFERS_SIZE (AIBUF_LEN * sizeof(s16) * ARRAY_COUNT(gAudioContext.aiBuffers))
+#define SOUNDFONT_INFO_SIZE (NUM_SOUNDFONTS * sizeof(SoundFont))
+#define PERMANENT_POOL_SIZE (SFX_SEQ_SIZE + SFX_SOUNDFONT_1_SIZE + SFX_SOUNDFONT_2_SIZE)
+
+const AudioHeapInitSizes audioHeapInitSizes = {
```
Comment (Dragorn421) [inline src/code/audio_init_params.c:24]:
```suggestion
const AudioHeapInitSizes gAudioHeapInitSizes = {
```
how could you :japanese_goblin:

File: src/code/audio_init_params.c [hunk 2]
```diff
@@ -10,79 +10,97 @@ const s16 D_8014A6C0[] = {
...
+    ALIGN16(sizeof(gAudioHeap) - 0x100),                                  // audio heap size
+    ALIGN16(PERMANENT_POOL_SIZE + AI_BUFFERS_SIZE + SOUNDFONT_INFO_SIZE), // init pool size
+    ALIGN16(PERMANENT_POOL_SIZE),                                         // permanent pool size
+};
+
+#define DEFAULT_REVERB_SETTINGS \
+    { 1, 0x30, 0x3000, 0, 0, 0x7FFF, 0x0000, 0x0000, 0xFF, 0x3000, 0x0, 0x0 }
+
+ReverbSettings reverbSettings[][3] = {
```
Comment (Dragorn421) [inline src/code/audio_init_params.c:33]:
```suggestion
ReverbSettings sReverbSettings[][3] = {
```

**Review by Dragorn421 — COMMENTED** [review]
Shameless plug for the diagram I made of the pools (`AudioHeapInitSizes` being about the root and red part):

<details>

audio heap pools layout

</details>

File: src/code/audio_init_params.c (hunk 1 above)
Comment (engineer124) [reply src/code/audio_init_params.c:24]: Whoops

**Review by Dragorn421 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**